### DISCUSSION TITLE
i found a funny behaviour

### DISCUSSION THREAD

**Discussion by to-json** [postmodern/ruby-install#449] [General]
i was playing with graalvm stuff, and noticed trufflegraal, and wanted to try it. this ended up being my first ruby-install installed ruby on this system. this made it my 'base ruby' for installing ...base, ruby, which i did as soon as i realized graal was my base ruby. this then failed; at the linker step, because the linker ruby script wants an 'arch' param? i worked around it by installing mruby, and then tried to use the 'delete' function, which fails because https://raw.githubusercontent.com/postmodern/ruby-versions/master/delete-ruby/versions.txt doesn't exist (latest released version, apparently from last week?)

i'll work around it locally, but it seems like a thing to mention; i can dig a bit more if that's useful.

**Comment by to-json** [comment]
oh, macos ventura, recent clang, none of that should be super important to the report, but, one is supposed to include it

**Comment by postmodern** [comment]
@to-json there is no delete function/sub-command, yet; I still need to add an uninstall option or `ruby-uninstall` command. To uninstall an installed ruby just delete it's installation directory, or if the ruby supports an uninstall task (ex: `make uninstall`) try running that in it's source directory (usually in `~/src/`). `ruby-install` supports installing `truffleruby` and `truffleruby-graalvm`, so I'm guessing you want `truffleruby-graalvm`. You shouldn't need to install a base ruby, since macOS still ships with ruby 2.6. If you need to install a newer version of CRuby, try `sudo ruby-install --system ruby` which will install into `/usr/local/`.

**Comment by postmodern** [comment]
You are correct that `ruby-install` should print an error when given a random ruby name:

```
$ ruby-install foo
>>> Updating foo versions ...
!!! Failed to download https://raw.githubusercontent.com/postmodern/ruby-versions/master/foo/versions.txt to /home/postmodern/.cache/ruby-install/foo/versions.txt!
!!! Failed to download foo versions!
```

**Comment by postmodern** [comment]
This should be fixed in 0.9.1 now:

```shell
$ ruby-install foo
ruby-install: unknown ruby: foo
```
